e2e/src/page-objects/pages/import-account.page.ts

```diff
@@ -0,0 +1,55 @@
+import { ImportAccountTestIds } from '../../../../src/app/pages/ImportAccount/ImportAccount.test-ids';
+import { Page } from '../../classes/page.class';
+import { createPageElement, findElements } from '../../utils/search.utils';
+
+export class ImportAccountTab extends Page {
+  tabSwitcher = createPageElement(ImportAccountTestIds.tabSwitcher);
+
+  async isVisible() {
+    await this.tabSwitcher.waitForDisplayed();
+  }
+}
+
+export class ImportAccountPrivateKeyTab extends Page {
```

Comment:
maybe create `pages/import-account` shared folder and split these page objects into separate files?

Reply:
Import accounts tabs are separated